app/helpers/status.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+const statusMap = {
+    pending: 'bg-warning',
+    accepted: 'bg-info',
+    implemented: 'bg-success',
+    rejected: 'bg-danger'
+};
+
+export function status(status) {
+    return statusMap[status];
```

Comment:
This would be better suited as a computed on the component/controller unless it's being used in many placed.

Also if someone has registered as a direct provider could `accepted` be shown to them as green?